tests/blockchain/eth/entities/deployers/test_worklock_deployer.py

```diff
@@ -40,11 +46,24 @@ def worklock_deployer(staking_escrow_deployer,
     return worklock_deployer
 
 
-def test_worklock_deployment(worklock_deployer, staking_escrow_deployer, deployment_progress):
+def test_worklock_deployment(worklock_deployer,
+                             baseline_deployment,
+                             staking_escrow_deployer,
+                             deployment_progress,
+                             test_registry,
+                             testerchain):
+
+    # Ensure nucypher APIs implementing economics are usable without a worklock deployment.
+    economics = EconomicsFactory.retrieve_from_blockchain(registry=test_registry)
+    assert economics.bidding_start_date == NotImplemented
 
+    # Deploy
     assert worklock_deployer.contract_name == WORKLOCK_CONTRACT_NAME
+    deployment_receipts = worklock_deployer.deploy(progress=deployment_progress)    # < ---- DEPLOY
```

Comment:
maybe assert something about the deployment receipts? The usual check on `status`

Reply:
Out of order, but the check is below.